### DISCUSSION TITLE
Add multi-region support

### DISCUSSION THREAD

**Discussion by brotbert** [superwerker/superwerker#197] [Ideas]
# Feature Request

I’ve noticed that Security Hub and GuardDuty are rolled out to the Control Tower home/main-region only.

Since both are regional services, is it planned to extend this to all Control Tower governed regions?

## Example use-case

Example use-case is running workloads in EU, but for several reasons it is required to split the workloads between

* eu-central-1
* eu-west-1
* us-east-1

## Why eu-west-1 over eu-central-1?

We (currently) would need to use eu-west-1 for following services:

* https://aws.amazon.com/iot-things-graph/
* https://aws.amazon.com/proton/
* https://aws.amazon.com/comprehend/medical/
* https://aws.amazon.com/devops-guru/
* https://aws.amazon.com/machine-learning/elastic-inference/
* https://aws.amazon.com/elastictranscoder/
* https://aws.amazon.com/fraud-detector/
* https://aws.amazon.com/ivs/
* https://aws.amazon.com/kendra/
* https://aws.amazon.com/location/
* https://aws.amazon.com/managed-blockchain/

## Why us-east-1?

* [CloudFront with ACM](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/cnames-and-https-requirements.html#https-requirements-aws-region)
* [CloudFront Triggers](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-requirements-limits.html#lambda-requirements-cloudfront-triggers)

**Comment by rzamana** [comment]
I don't think a "why" is required... There is possibility to have workloads in multiple regions, and is not possible to easily maintain/deploy the security features on a multi region Landing Zone.
